### PR TITLE
Adding handleReturnItem method in ReturnsMethods class for handling return items. We are now working with v3 for returns api as well. So the content-type is fixed for v3

### PR DESCRIPTION
**Pull request by bharti-dreambits**
…eturn items

### CODE REVIEW

File: bol/retailer/api.py [hunk 1]
```diff
@@ -260,9 +260,14 @@ def get(self, page=1):
         return ReturnItems.parse(self.api, resp.text)
 
     def getSingle(self, rmaId):
-        resp = self.request("GET", path=rmaId)
+        resp = self.request("GET", path=str(rmaId))
         return SingleReturnItem.parse(self.api, resp.text)
 
+    def handleReturnItem(self, return_no, status_reason, qty):
```
Comment (karan-dreambits) [inline bol/retailer/api.py:266]: change return_no to rmaId

File: bol/retailer/api.py [hunk 2]
```diff
@@ -359,11 +364,7 @@ def request(self, method, uri, params={}, **kwargs):
             # If these headers are not added, the api returns a 400
             # Reference:
             #   https://api.bol.com/retailer/public/conventions/index.html
-            content_header = ""
-            if "returns" in uri:
-                content_header = "application/vnd.retailer.v4+json"
-            else:
-                content_header = "application/vnd.retailer.v3+json"
+            content_header = "application/vnd.retailer.v3+json"
```
Comment (karan-dreambits) [inline bol/retailer/api.py:367]: This change is necessary but the commit needs to give explaination that why this is changed.
Earlier it was set as v3 only we changed to try if we can work with v4 version with just returns api

**Review by karan-dreambits — CHANGES_REQUESTED** [review]

File: bol/retailer/api.py (hunk 1 above)
Comment (bharti-dreambits) [reply bol/retailer/api.py:266]: Ok sir

File: bol/retailer/api.py (hunk 2 above)
Comment (bharti-dreambits) [reply bol/retailer/api.py:367]: Ok sir ,
Please let me know if the below lines will work for the same :

"We are know working with v3 for returns api as well. So the content-type is fixed for v3"

**Pull request merged**